lib/server.js

```diff
@@ -39,6 +39,7 @@ var Server = function (server, path, services, wsdl, options) {
   this.wsdl = wsdl;
   this.suppressStack = options && options.suppressStack;
   this.returnFault = options && options.returnFault;
+  this.onewayOptions = options && options.oneWay;
```

Comment:
Why not add ` || {}` here so `onewayOptions` is _always_ an object?  Should reduce typing elsewhere.

Reply:
Please change this to `this.onewayOptions = options && options.oneWay || {};`.  We can follow up later to change existing style.

Further down in this file, you can assume it'll always be an object.